server/src/main/java/org/eclipse/openvsx/RegistryAPI.java

```diff
@@ -1713,4 +1713,9 @@ public ResponseEntity<String> getPublicKey(
 
         return ResponseEntity.notFound().build();
     }
+
+    @GetMapping(path = "/api/version", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<RegistryVersionJson> getServerVersion() {
+        return ResponseEntity.ok(local.getRegistryVersion());
```

Comment:
I see other methods are catching the exceptions and then process it something like this:

```java
...
        } catch (ErrorResultException exc) {
            return exc.toResponseEntity(RegistryEntityJson.class);
        }
```

Should it be in the same way here?

Reply:
Right, this seems to be the pattern we should be following. I adjusted the method accordingly.